### PR TITLE
BUG ensures event_name proceeded by the problem_name

### PR DESCRIPTION
**Pull request by maikia**
#### Reference Issue

Fixes: #330

The event_name could have been freely chosen when the event was created at the terminal level.
However, when updated from the frontend the event was ensured to have the problem_name as the prefix which sometimes was leading to unexpected change of name of the event

It now checks if the event_name has the problem_name as the prefix set in the config.yml at the creation of the event. if not it throws an error

@tomMoral @agramfort @lucyleeow

### CODE REVIEW

**Comment by agramfort** [comment]
good to go?

File: doc/create_ramp_event.rst [hunk 1]
```diff
@@ -42,7 +42,9 @@ This config file should look like::
         conda_env: ramp-iris
 
 .. note::
-    In the previous configuration example, we are using a conda worker. You can
+    - <event_name must always be preceeded with the '<problem_name>_' as in
```
Comment (tomMoral) [inline doc/create_ramp_event.rst]: `<event_name` -> Typo?

File: ramp-frontend/ramp_frontend/tests/test_ramp.py [hunk 2]
```diff
@@ -151,24 +152,47 @@ def test_problem(client_session):
...
+    client, session = client_session
+    if not correct:
+        with pytest.raises(ValueError) as e:
+            add_event(session, 'iris', event_name, 'new_event', 'starting_kit',
+                      '/tmp/databoard_test/submissions', is_public=True)
+        assert "<event_name> must start with" in str(e.value)
+    else:
+        assert add_event(session, 'iris', event_name, 'new_event',
+                         'starting_kit', '/tmp/databoard_test/submissions',
+                         is_public=True)
```
Comment (tomMoral) [inline ramp-frontend/ramp_frontend/tests/test_ramp.py]: It is great to have a test!

**Comment by tomMoral** [comment]
Fine for me!

**Comment by agramfort** [comment]
Thx @maikia

**Comment by glemaitre** [comment]
Thanks @maikia

**Comment by maikia** [comment]
thanks @agramfort @glemaitre @tomMoral

**Pull request merged**